### DISCUSSION TITLE
Interact with Pluto in the terminal?

### DISCUSSION THREAD

**Discussion by Fainabi** [fonsp/Pluto.jl#1817] [Questions about using Pluto] [answered]
## Problem Met
I was using some other packages, and when constructing a dataset, I faced an asking in the terminal
![image](https://user-images.githubusercontent.com/52078284/148773570-ac50e72c-a6dc-4757-9e15-705aaa4f5c4c.png)

codes are like following:

```julia
begin
    # the outer packages imported using `PyCall.jl`
    T = pyimport("torch_geometric").transforms
    PygNodePropPredDataset = pyimport("ogb.nodeproppred").PygNodePropPredDataset
    dataset_name = "ogbn-arxiv"

    # prompted while running this script
    dataset = PygNodePropPredDataset(name=dataset_name, transform=T.ToSparseTensor())
end
```

I don't know how to enter my decision, therefore all the other following cells were blocked in the notebook.

I've tried using `@with_terminal`, but didn't make it.

## Question
So my question is that is it possible to enter something to `stdin` in the terminal, or with other native `PlutoUI` elements?

**Comment by icweaver** [comment, marked as answer]
Hey @Fainabi! According to this [closed issue](https://github.com/fonsp/Pluto.jl/issues/1394), it doesn't look like support for this is available by design

**Reply by Fainabi** [reply]
Got it, thanks for your reply.
